app/services/deploy_status_checker.rb

```diff
@@ -5,7 +5,7 @@ class DeployStatusChecker
   Environment = Struct.new(:env, :statuses)
 
   DEPLOYS = [
-    Deploy.new('identity-idp', 'prod', 'https://idp.prod.login.gov'),
+    Deploy.new('identity-idp', 'prod', 'https://secure.login.gov'),
     Deploy.new('identity-sp-rails', 'prod'),
     Deploy.new('identity-sp-sinatra', 'prod'),
     Deploy.new('identity-dashboard', 'prod'),
```

Comment:
Without a URL we just show an empty box, I kept it to keep the layout consistent?

<img width="1086" alt="identity_dashboard" src="https://cloud.githubusercontent.com/assets/458784/24669228/a0a4c4ee-1937-11e7-9a58-7b1468482828.png">